database/insertData/laos/03_template_license_MMC_MMD.js

```diff
@@ -0,0 +1,1559 @@
+#                 colour: "#E17E48" #orange
+#                 templateStageReviewLevelsUsingId: {
+#                   create: [{ number: 1, name: "Assessor" }]
+#                 }
+#               }
+              ]
+            }
+            templateCategoryToTemplateCategoryId: {
+              connectByCode: { code: "license" }
+            }
+            ${joinFilters}
+            templateActionsUsingId: {
+              create: [
+                ${coreActions}
+                ${devActions}
+                {
+                  actionCode: "cLog"
+                  trigger: ON_APPLICATION_SUBMIT
+                  parameterQueries: {
+                    message: { value: "Organisation Registration submission" }
+                  }
+                }
+                {
+                  actionCode: "changeOutcome"
+                  trigger: ON_REVIEW_SUBMIT
+                  sequence: 100
+                  condition: {
+                    operator: "AND"
+                    children: [
+                      {
+                        operator: "="
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: [
+                              "applicationData.reviewData.latestDecision.decision"
+                            ]
+                          }
+                          "CONFORM"
+                        ]
+                      }
+                      {
+                        operator: "objectProperties"
+                        children: ["applicationData.reviewData.isLastLevel"]
+                      }
+                    ]
+                  }
+                  parameterQueries: { newOutcome: { value: "APPROVED" } }
+                }
+                # TODO: Generate the expiry and serial before this
+                {
+                  actionCode: "modifyRecord"
+                  trigger: ON_REVIEW_SUBMIT
+                  sequence: 101
+                  condition: {
+                    operator: "="
+                    children: [
+                      {
+                        operator: "objectProperties"
+                        children: ["applicationData.outcome"]
+                      }
+                      "APPROVED"
+                    ]
+                  }
+                  parameterQueries: {
+                    tableName: "license"
+                    type: {
+                      operator: "objectProperties"
+                      children: ["applicationData.responses.Q2LicenseType.text"]
+                    }
+                    expiry: {
+                      operator: "objectProperties"
+                      children: ["31/01/2022"]
+                    }
+                    company_id: {
+                      operator: "objectProperties"
+                      children: ["currentUser.orgId"]
+                    }
```

Comment:
Are you just trying to insert hard-coded values? This isn't working, but you can just put the literal values in:
```
expiry: "31/01/2022",
```

Reply:
Cool. Changing to that now : D